docs/content/routing/services/index.md

```diff
@@ -1081,6 +1081,86 @@ http:
         url = "http://private-ip-server-2/"
 ```
 
+#### FailoverService
+
+Failover service allows to reference a service that will be use when no servers are available on the loadbalancer.
+
+!!! info "This is having effects only when `healthCheck` is set on children"
```

Comment:
```suggestion
!!! info "A failover service can only take over when all the children of the initial service are detected as unavailable. This can only happen if `healthCheck` is enabled for all the children."
```